src/main/scala/nl/knaw/dans/easy/bag2deposit/ddm/AbrRewriteRule.scala

```diff
@@ -21,43 +21,53 @@ import nl.knaw.dans.easy.bag2deposit.parseCsv
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 import org.apache.commons.csv.CSVRecord
 
+import scala.util.matching.Regex
 import scala.xml.transform.RewriteRule
 import scala.xml.{ Elem, Node }
 
-case class AbrRewriteRule(cfgFile: File, oldLabel: String, newLabel: String) extends RewriteRule with DebugEnhancedLogging {
-  private val map = parse(cfgFile, newLabel)
+case class AbrRewriteRule(cfgFile: File, oldLabel: String, newLabel: String, scheme: String, schemeURI: String) extends RewriteRule with DebugEnhancedLogging {
```

Comment:
With so many arguments we might be better of with two classes that at least override `toXml`.